<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2399</width>
    <height>1694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="global_horiziontal" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="left" stretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QGridLayout" name="left_grid">
         <item row="22" column="0">
          <widget class="QLabel" name="l_det_number">
           <property name="text">
            <string>det_number</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="l_gamma">
           <property name="text">
            <string>gamma</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="l_c">
           <property name="text">
            <string>c</string>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="l_det_rot">
           <property name="text">
            <string>det_rot</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="l_alpha">
           <property name="text">
            <string>alpha</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lE_Spacegroup">
           <property name="text">
            <string>P 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="l_hkl2_p">
           <property name="text">
            <string>hkl2_p</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="25" column="0">
          <widget class="QLabel" name="l_omega_offset">
           <property name="text">
            <string>omega offset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="l_cif_file">
           <property name="text">
            <string>CIF File</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="l_b">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QLabel" name="l_sample_rot">
           <property name="text">
            <string>sample_rot</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="l_a">
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="dSB_wavelength">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>4.200000000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="l_spacegroup">
           <property name="text">
            <string>Spacegroup</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="2">
          <widget class="QLabel" name="l_vectors">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Vectors in scattering plane</string>
           </property>
          </widget>
         </item>
         <item row="25" column="1">
          <widget class="QDoubleSpinBox" name="dSB_omega_offset">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-720.000000000000000</double>
           </property>
           <property name="maximum">
            <double>720.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="lE_hkl2_p">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QDoubleSpinBox" name="dSB_det_rot">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-30.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>-5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QCheckBox" name="cB_unique">
           <property name="text">
            <string>show only unique</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pB_load_cif">
           <property name="text">
            <string>load cif</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pB_clear_cif">
           <property name="text">
            <string>clear cif</string>
           </property>
          </widget>
         </item>
         <item row="24" column="0" colspan="2">
          <widget class="QCheckBox" name="cB_fix_omega">
           <property name="text">
            <string>fix omega offset</string>
           </property>
          </widget>
         </item>
         <item row="20" column="0" colspan="2">
          <widget class="QLabel" name="l_idendity">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Identify:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="l_hkl2">
           <property name="text">
            <string>hkl2</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="l_hkl1">
           <property name="text">
            <string>hkl1</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="l_beta">
           <property name="text">
            <string>beta</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="l_wavelength">
           <property name="text">
            <string>Wavelength</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="dSB_a">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="l_kival">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="dSB_b">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="dSB_c">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="l_ki">
           <property name="text">
            <string>k_i</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="lE_hkl2">
           <property name="text">
            <string>1,1,1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="0" colspan="2">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="17" column="0" colspan="2">
          <widget class="QCheckBox" name="cB_inplane">
           <property name="text">
            <string>show only inplane</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="lE_hkl1">
           <property name="text">
            <string>1,1,0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lE_cif_filename">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="23" column="1">
          <widget class="QDoubleSpinBox" name="dSB_sample_rot">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="dSB_gamma">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="dSB_alpha">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="dSB_beta">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="22" column="1">
          <widget class="QSpinBox" name="SB_det_number">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pB">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
